packages/analytics-browser/src/attribution/campaign-parser.ts

```diff
@@ -0,0 +1,76 @@
+import { UTMCookie } from '../storage/utm-cookie';
```

Comment:
dedicated module to parse all campaign related data from url params, cookies, document